src/content/developers/docs/scaling/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Scaling
+description: An introduction to the different scaling options currently being developed by the Ethereum community.
+lang: en
+sidebar: true
+incomplete: true
+---
+
+## Scaling Overview {#scaling-overview}
+
+As the number of people using Ethereum has grown, the blockchain in its native form has reached certain capacity limitations. This has driven up the cost of using the network, creating the need for "scaling solutions." There are multiple solutions being researched, tested and implemented that take different approaches to achieving similar goals.
```

Comment:
```suggestion
As the number of people using Ethereum has grown, the blockchain has reached certain capacity limitations. This has driven up the cost of using the network, creating the need for "scaling solutions." There are multiple solutions being researched, tested and implemented that take different approaches to achieve similar goals.
```

Reply:
Suggestion to remove "in its native form" - not really sure what that means but feel free to clarify.